audio-worklet/src/bitcrusher-demo.js

```diff
@@ -32,23 +32,30 @@ class BitcrusherDemo {
       reduction: reduction || 1
     });
 
-    this.bitcrusher_.output.connect(this.gainNode_);
-    this.gainNode_.connect(this.context_.destination);
+    this.bitcrusher_.output.connect(this.masterGain_);
+    this.masterGain_.connect(this.context_.destination);
 
-    const songUrl = 'audio/it-from-bit.mp3';
+    this.loadSong_('audio/it-from-bit.mp3', (song) => {
+      this.songBuffer = song;
+      this.sourceButton_.enable()
+    });
+  }
 
-    // Fetch song from server and decode into buffer.
-    fetch(songUrl).then(response => response.arrayBuffer().then(song => {
-      this.context_.decodeAudioData(song).then(buffer => {
-        this.songBuffer = buffer;
-        this.sourceButton_.enable();
-      });
-    }));
+  async loadSong_(url, enableSong) {
+    try {
+      const response = await fetch(url);
+      const song = await response.arrayBuffer();
+      const buffer = await this.context_.decodeAudioData(song);
+      enableSong(buffer);
+    }
+    catch (err) {
+      console.log('Could not retrieve song.', err);
```

Comment:
Shouldn't this be an error and stop the demo?  The demo is basically broken if you can't get the song to play.